wp-content/plugins/mrb-jobs-feed/mrb-jobs-feed.php

```diff
@@ -19,8 +19,19 @@
 // Plugin Autoloader
 require_once __DIR__.'/autoloader.php';
 
+
 // Grab the core Plugin class
 $core = Core::class;
 
 //  Initialise
 add_action('plugins_loaded', [$core, 'initialise']);
+
+
+
+	//add js
+		add_action('wp_enqueue_scripts',[$core,'my_js_files']);
```

Comment:
I'm curious as to why you've elected to add actions/filter here? What other options are there and how do they compare to your chosen route?